react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -793,16 +792,11 @@ class Thumbnail extends Component<Props, State> {
                 <div className = 'videocontainer__toptoolbar'>
                     { this._renderTopIndicators() }
                 </div>
-                <div className = 'videocontainer__hoverOverlay' />
-                <div
-                    className = 'displayNameContainer'
-                    onClick = { onClick }>
-                    <DisplayName
-                        allowEditing = { !_disableProfile }
-                        displayNameSuffix = { _defaultLocalDisplayName }
-                        elementID = 'localDisplayName'
-                        participantID = { id } />
-                </div>
+                { _isTileView && (
```

Comment:
Does that mean that in stage view you won't be able to see the name of the participant? Is this intentional?

Reply:
Stage view will have a badge displayed on top of the toolbar (DominantSpeakerName), whereas thumbnails will have this component present within the actual thumbnail.

Reply:
But with this check the thumbnails will have the component present only in tile view. Isn't this correct? Am I missing something?

I tested stage view and I don't see a way to see the display name of a participant.